{"abstract":[{"text":"Happens when there is an error caused by something on the device itself","type":"text"}],"metadata":{"modules":[{"name":"IdVerification365id"}],"roleHeading":"Case","symbolKind":"case","title":"IdVerificationError.clientError","role":"symbol","fragments":[{"text":"case","kind":"keyword"},{"kind":"text","text":" "},{"kind":"identifier","text":"clientError"}],"externalID":"c:@M@IdVerification365id@E@IdVerificationError@IdVerificationErrorClientError"},"kind":"symbol","identifier":{"url":"doc:\/\/com.365id.sdk.idverification\/documentation\/IdVerification365id\/IdVerificationError\/clientError","interfaceLanguage":"swift"},"hierarchy":{"paths":[["doc:\/\/com.365id.sdk.idverification\/documentation\/IdVerification365id","doc:\/\/com.365id.sdk.idverification\/documentation\/IdVerification365id\/IdVerificationError"]]},"variants":[{"paths":["\/documentation\/idverification365id\/idverificationerror\/clienterror"],"traits":[{"interfaceLanguage":"swift"}]}],"sections":[],"schemaVersion":{"minor":3,"major":0,"patch":0},"primaryContentSections":[{"declarations":[{"languages":["swift"],"tokens":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"text":"clientError","kind":"identifier"}],"platforms":["iOS"]}],"kind":"declarations"}],"references":{"doc://com.365id.sdk.idverification/documentation/IdVerification365id/IdVerificationError/clientError":{"url":"\/documentation\/idverification365id\/idverificationerror\/clienterror","identifier":"doc:\/\/com.365id.sdk.idverification\/documentation\/IdVerification365id\/IdVerificationError\/clientError","title":"IdVerificationError.clientError","fragments":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"kind":"identifier","text":"clientError"}],"abstract":[{"text":"Happens when there is an error caused by something on the device itself","type":"text"}],"role":"symbol","kind":"symbol","type":"topic"},"doc://com.365id.sdk.idverification/documentation/IdVerification365id":{"title":"IdVerification365id","identifier":"doc:\/\/com.365id.sdk.idverification\/documentation\/IdVerification365id","abstract":[],"type":"topic","url":"\/documentation\/idverification365id","kind":"symbol","role":"collection"},"doc://com.365id.sdk.idverification/documentation/IdVerification365id/IdVerificationError":{"navigatorTitle":[{"text":"IdVerificationError","kind":"identifier"}],"title":"IdVerificationError","type":"topic","url":"\/documentation\/idverification365id\/idverificationerror","fragments":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"IdVerificationError"}],"abstract":[{"text":"Id Verification Errors used in the ","type":"text"},{"code":"onError(error:)","type":"codeVoice"},{"type":"text","text":" method in the "},{"type":"codeVoice","code":"IdVerificationEventDelegate"}],"identifier":"doc:\/\/com.365id.sdk.idverification\/documentation\/IdVerification365id\/IdVerificationError","role":"symbol","kind":"symbol"}}}